string/global regex matcher/8449789e-3b23-4bf5-9c33-931015049dc5.json

```diff
@@ -1,14 +1,16 @@
 {
-  "description": "Executes a global regex pattern match",
+  "description": "Executes a global regex pattern match.\n\nFlavour depends on the running environment.",
```

Comment:
Not sure if this additional sentence is comprehensible enough for a user to assess the eventual consequences. In fact not even for me...